manifest.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version='1.0' encoding='utf-8' ?>
+<ExtensionImage xmlns="http://schemas.microsoft.com/windowsazure">
+  <ProviderNameSpace>Microsoft.Azure.Extensions</ProviderNameSpace>
```

Comment:
The big ones are an SPN and a storage account with an extensions container that's publically accessible.

It's fine to move away from it, but you're going to make it harder to upgrade.  autoUpgradeMinorVersion is no longer applicable, so customers do not have a simple upgrade path to the newer versions.  It's fine if you want to be _2.0_, but it will not work if you want to be _1.6_.